libMBIN/Source/Template/NMSTemplate.cs

```diff
@@ -688,13 +695,6 @@ public void SerializeGenericList( BinaryWriter writer, IList list, long listHead
 
             foreach ( var entry in list ) {
                 DebugLogTemplate( $"[C] writing {entry.GetType().Name} to offset 0x{writer.BaseStream.Position:X}" );
-                int alignment = entry.GetType().GetCustomAttribute<NMSAttribute>()?.Alignment ?? 0x4;
```

Comment:
this is being removed because it gets handled by lines 663 onward right?

Reply:
Thei is being removed because alignment gets handled again in SerializeValue().
The stuff from lines 663 onward is only needed to get the correct file offset.